app/assets/stylesheets/main.scss

```diff
@@ -34,3 +34,7 @@ footer {
   top: 0;
   width: 100%;
 }
+
+#containner {
```

Comment:
`containner`? it's seems, shoild be `container`

Reply:
It's already redundant